contracts/TransactionRequest.sol

```diff
@@ -73,19 +77,38 @@ contract TransactionRequest is Digger, TransactionRequestInterface {
                 txnRequest.serializedValues.uint8Values
             );
         } else {
-            throw;
+            revert();
         }
     }
 
-    function callData() constant returns (bytes) {
-        return txnRequest.txnData.callData;
+    function callData() 
+        public view returns (bytes data)
+    {
+        data = txnRequest.txnData.callData;
+    }
+
+    /**
+     * @dev Proxy a call from this contract to another contract.
+     * This function is only callable by the scheduler and can only
+     * be called after the execution window ends. It's purpose is to
+     * provide a way to transfer assets held by this contract somewhere else.
+     * For example, if this request was used to buy tokens during an ICO,
+     * it would become the owner of the tokens and this function would need
+     * to be called with the encoded data to the token contract to transfer
+     * the assets somewhere else. */
+    function proxy(address _to, bytes _data)
+        public payable returns (bool success)
+    {
+        require(txnRequest.meta.owner == msg.sender
+                && txnRequest.schedule.isAfterWindow());
+        return _to.call.value(msg.value)(_data);
     }
 
     /*
      *  Pull based payment functions.
      */
-    function refundClaimDeposit() public returns (bool) {
-        return txnRequest.refundClaimDeposit();
+    function refundClaimDeposit() public {
```

Comment:
Any reason the return value was removed from this?

Reply:
I changed it to revert/throw if it returned false. But I think we prefer explicit returns instead so I added the return value back.